src/main/java/de/craften/ui/swingmaterial/MaterialWindow.java

```diff
@@ -8,29 +8,216 @@
+        if (Utils.isTranslucencySupported()) {
             setBackground(new Color(255, 255, 255, 0));
-            setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-            setContentPane(new ShadowPane());
-            getRootPane().putClientProperty("Window.shadow", Boolean.FALSE);
-            getRootPane().setWindowDecorationStyle(JRootPane.NONE);
-            addComponentListener(new ComponentAdapter() {
-                @Override
-                public void componentResized(ComponentEvent componentEvent) {
-                    setShape(new RoundRectangle2D.Float(0, 0, getWidth(), getHeight(), 3, 3));
-                }
-            });
+            //This default behavior should NOT be set here!
+            //setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+            super.setContentPane((shadowPane = new ShadowPane()));
+            contentPane = new JPanel();
+            shadowPane.add(contentPane);
+        } else {
+            contentPane = super.getContentPane();
         }
+        getRootPane().putClientProperty("Window.shadow", Boolean.FALSE);
+        getRootPane().setWindowDecorationStyle(JRootPane.NONE);
+        addComponentListener(new ComponentAdapter() {
+            @Override
+            public void componentResized(ComponentEvent componentEvent) {
+                setShape(new RoundRectangle2D.Float(0, 0, getWidth(), getHeight(), 3, 3));
+            }
+        });
+        light();
     }
 
-    private class ShadowPane extends JPanel {
+    @Override
+    public Container getContentPane() {
+        return contentPane;
+    }
+
+    @Override
+    public void setContentPane(Container contentPane) {
+        this.contentPane = contentPane;
+    }
+    
+    /**
+     * Chainable method to quickly get a {@link MaterialWindow} with a dark
+     * background and white foreground. You can append it to your constructor
+     * like this: {@code MaterialWindow myWindow = new MaterialWindow.dark()} or
+     * just use it alone to switch the foreground/background colors.
+     * @return this window
+     */
+    public MaterialWindow dark() {
+        getContentPane().setBackground(MaterialColor.GREY_900);
+        getContentPane().setForeground(MaterialColor.WHITE);
+        return this;
+    }
+    
+    /**
+     * Chainable method to quickly get a {@link MaterialWindow} with a white
+     * background and black foreground. You can append it to your constructor
+     * like this: {@code MaterialWindow myWindow = new MaterialWindow.dark()} or
+     * just use it alone to switch the foreground/background colors.
+     * @return this window
+     */
+    public MaterialWindow light() {
+        getContentPane().setBackground(MaterialColor.WHITE);
+        getContentPane().setForeground(MaterialColor.GREY_900);
+        return this;
+    }
+    
+    /**
+     * Alternative method to {@link JFrame#setLocation(int, int)}. Use it to set
+     * a new location on screen for this window, this method takes into
```

Comment:
Maybe we should override `JFrame#setLocation`, just call `super.setLocation` and deprecate it and tell the user to use this instead to guide users of this to using this method.